aptos-move/framework/aptos-framework/sources/Token.move

```diff
@@ -314,17 +361,90 @@ module AptosFramework::Token {
         &token.id
     }
 
-    /// Transfers `amount` of tokens from `from` to `to`.
-    public fun transfer(
+    /// Transfer one token
+    public(script) fun transferFrom(
         from: &signer,
-        id: &TokenId,
         to: address,
+        id: &TokenId
+    ) acquires TokenStore {
+        let token = withdraw_token(from, id ,1);
+        direct_deposit(to, token);
+    }
+
+    /// Transfers `amount` of tokens
+    public(script) fun transferFromMultiple(
+        from: &signer,
+        to: address,
+        id: &TokenId,
```

Comment:
I don't think there's a need to distinguish here. Wallets and DApps will be writing the transactions, not users directly.

Reply:
the ERC721 use transferFrom as the method name. I am trying make our naming more consistent. Anyway. I revert the change.  we can discuss later.